app/bundles/LeadBundle/Model/LeadModel.php

```diff
@@ -2345,7 +2345,10 @@ private function validateSelectFields(Lead $entity, ?array $fields): void
                 $allowedValues = is_array($field['properties'])
                     ? $field['properties']
                     : unserialize($field['properties']);
-                $flattenedAllowedValues = array_column($allowedValues['list'], 'value');
+
+                $flattenedAllowedValues = array_map(function ($item) {
+                    return html_entity_decode($item['value'], ENT_QUOTES);
+                }, $allowedValues['list']);
```

Comment:
@nileshlohar since Rector is failing on this anyway, I'd like to suggest an arrow function
```suggestion
                $flattenedAllowedValues = array_map(fn ($item): string => html_entity_decode($item['value'], ENT_QUOTES), $allowedValues['list']);
```

Reply:
Thanks @escopecz. I have fixed it.